cypress/e2e/smoke/legal.cy.js

```diff
@@ -0,0 +1,22 @@
+import '@testing-library/cypress/add-commands'
+
+describe('Legal page', () => {
+
+  beforeEach(() => {
```

Comment:
We can remove the `beforeEach` if no longer needed
```suggestion
  beforeEach(() => {
```